tfx/dsl/component/experimental/placeholders.py

```diff
@@ -55,6 +67,12 @@ class OutputUriPlaceholder(json_utils.Jsonable):
   def __init__(self, output_name: Text):
     self.output_name = output_name
 
+  def __eq__(self, other):
+    return (isinstance(other, self.__class__) and
+           self.output_name == other.output_name)
+
+  def __ne__(self, other):
+    return not self.__eq__(other)
 
 class ConcatPlaceholder(object):
```

Comment:
Sorry this is out of scope of this PR: do we know why this is not `Jsonable`?

/cc @charlesccychen

Reply:
It should be. The CLs adding ConcatPlacholder and making existing placeholders Jsonable were parallel.